Index: mame/scripts/src/osd/sdl_cfg.lua
===================================================================
--- mame.orig/scripts/src/osd/sdl_cfg.lua	2015-05-28 06:45:50.064834000 +0200
+++ mame/scripts/src/osd/sdl_cfg.lua	2015-05-28 06:45:50.036834000 +0200
@@ -18,6 +18,11 @@
 	}
 end
 
+if _OPTIONS["SDL_INI_PATH"]~=nil then
+    defines {
+        "'INI_PATH=\"" .. _OPTIONS["SDL_INI_PATH"] .. "\"'",
+    }
+end
 
 if _OPTIONS["NO_X11"]=="1" then
 	defines {
Index: mame/makefile
===================================================================
--- mame.orig/makefile	2015-05-28 06:45:50.064834000 +0200
+++ mame/makefile	2015-05-28 06:45:50.036834000 +0200
@@ -29,6 +29,7 @@
 # USE_DISPATCH_GL = 0
 # DIRECTINPUT = 7
 # USE_SDL = 1
+# SDL_INI_PATH = .;$HOME/.mame/;ini;
 # SDL2_MULTIAPI = 1
 # NO_USE_MIDI = 1
 # DONT_USE_NETWORK = 1
@@ -467,6 +468,10 @@
 PARAMS += --USE_SDL='$(USE_SDL)'
 endif
 
+ifdef SDL_INI_PATH
+PARAMS += --SDL_INI_PATH='$(SDL_INI_PATH)'
+endif
+
 ifdef CYGWIN_BUILD
 PARAMS += --CYGWIN_BUILD='$(CYGWIN_BUILD)'
 endif
Index: mame/scripts/src/osd/sdl.lua
===================================================================
--- mame.orig/scripts/src/osd/sdl.lua	2015-05-28 06:45:50.064834000 +0200
+++ mame/scripts/src/osd/sdl.lua	2015-05-28 06:45:50.040834000 +0200
@@ -102,6 +102,11 @@
 }
 
 newoption {
+    trigger = "SDL_INI_PATH",
+    description = "Default search path for .ini files",
+}
+
+newoption {
 	trigger = "NO_X11",
 	description = "Disable use of X11",
 	allowed = {
