commit 689c19720ae7854ec50024972ae8ab1232837d15
Author: Vas Crabb <vas@vastheman.com>
Date:   Sat May 14 16:50:14 2016 +1000

    Hopefully fix PPC targets

diff --git a/src/emu/video/rgbvmx.h b/src/emu/video/rgbvmx.h
index f5829c1..95a7d45 100644
--- a/src/emu/video/rgbvmx.h
+++ b/src/emu/video/rgbvmx.h
@@ -28,11 +28,11 @@ protected:
 	typedef __vector unsigned int   VECU32;
 
 public:
-	inline rgbaint_t(): VECS8(0), VECU8(0), VECS16(0), VECU16(0), VECS32(0), VECU32(0) { }
+	inline rgbaint_t() { set(0, 0, 0, 0); }
 	inline rgbaint_t(UINT32 rgba) { set(rgba); }
 	inline rgbaint_t(INT32 a, INT32 r, INT32 g, INT32 b) { set(a, r, g, b); }
 	inline rgbaint_t(rgb_t& rgb) { set(rgb); }
-	inline rgbaint_t(VECS32 rgba): VECS8(0), VECU8(0), VECS16(0), VECU16(0), VECS32(0), VECU32(0) { m_value = rgba; }
+	inline rgbaint_t(VECS32 rgba) : m_value(rgba) { }
 
 	inline void set(rgbaint_t& other) { m_value = other.m_value; }
 
