Description: build with system libjpeg8
Author: Emmanuel Kasper <emmanuel@libera.cc>
Forwarded: Yes

--- a/makefile
+++ b/makefile
@@ -628,7 +628,8 @@
 # add formats emulation library
 FORMATS_LIB = $(OBJ)/libformats.a
 
-JPEG_LIB = $(OBJ)/libjpeg.a
+# Link with system jpeg
+LIBS += -ljpeg
 
 #-------------------------------------------------
 # 'default' target needs to go here, before the 
--- a/src/mame/drivers/sliver.c
+++ b/src/mame/drivers/sliver.c
@@ -67,7 +67,13 @@
 #include "sound/okim6295.h"
 #include "cpu/mcs51/mcs51.h"
 #include "video/ramdac.h"
-#include "../../lib/libjpeg/jpeglib.h"
+
+/* This should prevent libjpeg of typedef'ing INT32 */
+#ifndef XMD_H
+#  define XMD_H
+#endif
+
+#include <jpeglib.h>
 
 
 #define FIFO_SIZE 1024
