Author: Cesare Falco <c.falco@ubuntu.com>
Description: Fix checking gcc version for Ubuntu 12.10 in makefile
Forwarded: yes

Index: mame-0.148/src/osd/sdl/sdl.mak
===================================================================
--- mame-0.148.orig/src/osd/sdl/sdl.mak	2013-01-11 08:32:46.000000000 +0100
+++ mame-0.148/src/osd/sdl/sdl.mak	2013-01-14 00:44:18.000000000 +0100
@@ -444,7 +444,7 @@
 ifeq ($(findstring 4.7.2-2ubuntu1,$(TEST_GCC)),4.7.2-2ubuntu1)
 GCC46TST = $(shell which g++-4.6 2>/dev/null)
 ifeq '$(GCC46TST)' ''
-	$(error Ubuntu 12.10 detected.  Please install the gcc-4.6 and g++-4.6 packages)
+$(error Ubuntu 12.10 detected.  Please install the gcc-4.6 and g++-4.6 packages)
 endif
 CC = @gcc-4.6
 LD = @g++-4.6
