From: Cesare Falco <cesare.falco@gmail.com>
Subject: Fix typos in mame tree to make lintian happier
Index: mame-0.142/src/emu/cpu/uml.c
===================================================================
--- mame-0.142.orig/src/emu/cpu/uml.c	2011-06-16 09:22:39.000000000 +0200
+++ mame-0.142/src/emu/cpu/uml.c	2011-06-16 09:33:13.000000000 +0200
@@ -921,7 +921,7 @@
 		if (pnum != 0)
 			buffer.cat(',');
 
-		// ouput based on type
+		// output based on type
 		switch (param.type())
 		{
 			// immediates have several special cases
Index: mame-0.142/src/lib/util/opresolv.c
===================================================================
--- mame-0.142.orig/src/lib/util/opresolv.c	2011-06-16 09:22:39.000000000 +0200
+++ mame-0.142/src/lib/util/opresolv.c	2011-06-16 09:33:13.000000000 +0200
@@ -552,7 +552,7 @@
 		"Parameter out of range",					/* OPTIONRESOLUTION_ERROR_PARAMOUTOFRANGE */
 		"Parameter not specified",					/* OPTIONRESOLUTION_ERROR_PARAMNOTSPECIFIED */
         "Unknown parameter",						/* OPTIONRESOLUTION_ERROR_PARAMNOTFOUND */
-		"Paramter specified multiple times",		/* OPTIONRESOLUTION_ERROR_PARAMALREADYSPECIFIED */
+		"Parameter specified multiple times",		/* OPTIONRESOLUTION_ERROR_PARAMALREADYSPECIFIED */
 		"Invalid parameter",						/* OPTIONRESOLUTION_ERROR_BADPARAM */
 		"Syntax error",								/* OPTIONRESOLUTION_ERROR_SYNTAX */
 		"Internal error"							/* OPTIONRESOLTUION_ERROR_INTERNAL */
Index: mame-0.142/src/mame/drivers/multfish.c
===================================================================
--- mame-0.142.orig/src/mame/drivers/multfish.c	2011-06-16 09:22:40.000000000 +0200
+++ mame-0.142/src/mame/drivers/multfish.c	2011-06-16 09:33:13.000000000 +0200
@@ -4081,7 +4081,7 @@
 GAME( 2005, islandb,     island_parent,   multfish, multfish,  customl,       ROT0,  "bootleg", "Island (bootleg, 050713, VIDEO GAME-1 OS01)", GAME_SUPPORTS_SAVE ) // custom alteras, modified graphics, many texts changed, changed version text to "VIDEO GAME-1 OS01"
 GAME( 2005, islandc,     island_parent,   multfish, multfish,  customl,       ROT0,  "bootleg", "Island (bootleg, 050713, LOTOS OS01)", GAME_SUPPORTS_SAVE ) // custom alteras, modified graphics, many texts changed, changed version text to "LOTOS OS01"
 
-GAME( 2006, island2a,    island2_parent,  island2a, multfish,  0,             ROT0,  "bootleg", "Island 2 (bootleg, 060529, banking address hack)", GAME_SUPPORTS_SAVE ) // bank F9 (not standart, game not work)
+GAME( 2006, island2a,    island2_parent,  island2a, multfish,  0,             ROT0,  "bootleg", "Island 2 (bootleg, 060529, banking address hack)", GAME_SUPPORTS_SAVE ) // bank F9 (not standard, game not work)
 GAME( 2006, island2b,    island2_parent,  multfish, multfish,  0,             ROT0,  "bootleg", "Island 2 (bootleg, 060529, banking address hack, changed version text)", GAME_SUPPORTS_SAVE ) // bank F9, changed version text to 070205, skip some start tests
 GAME( 2006, island2c,    island2_parent,  multfish, multfish,  0,             ROT0,  "bootleg", "Island 2 (bootleg, 060529, LOTTOGAME (I))", GAME_SUPPORTS_SAVE ) // bank F9, modified graphics, changed version text to "MDS_is_the_best_ LOTTOGAME (I)"
 GAME( 2006, island2_3a,  island2_parent,  multfish, multfish,  0,             ROT0,  "bootleg", "Island 2 (bootleg, 061218, VIDEO GAME-1 OS2-01)", GAME_SUPPORTS_SAVE ) // bank F9, modified graphics, changed version text to "VIDEO GAME-1 OS2-01"
Index: mame-0.142/src/mame/drivers/segaybd.c
===================================================================
--- mame-0.142.orig/src/mame/drivers/segaybd.c	2011-06-16 09:22:40.000000000 +0200
+++ mame-0.142/src/mame/drivers/segaybd.c	2011-06-16 09:33:13.000000000 +0200
@@ -2193,7 +2193,7 @@
 {
 	/* r360 cabinet */
 	output_set_value("start_lamp", BIT(data, 2));
-	/* even though the same ouput is used, I've split them to avoid confusion. */
+	/* even though the same output is used, I've split them to avoid confusion. */
 	output_set_value("emergency_stop_lamp", BIT(data, 2));
 }
 
Index: mame-0.142/src/mame/machine/atarigen.c
===================================================================
--- mame-0.142.orig/src/mame/machine/atarigen.c	2011-06-16 09:22:40.000000000 +0200
+++ mame-0.142/src/mame/machine/atarigen.c	2011-06-16 09:33:13.000000000 +0200
@@ -1609,7 +1609,7 @@
 	// remember the control state
 	m_earom_control = data;
 
-	// ensure ouput data is put on data lines prior to updating controls
+	// ensure output data is put on data lines prior to updating controls
 	if (m_earom_control & 4)
 		m_earom->set_data(m_earom_data);
 
Index: mame-0.142/src/mame/machine/kanekotb.h
===================================================================
--- mame-0.142.orig/src/mame/machine/kanekotb.h	2011-06-16 09:22:40.000000000 +0200
+++ mame-0.142/src/mame/machine/kanekotb.h	2011-06-16 09:33:13.000000000 +0200
@@ -11,7 +11,7 @@
 ***************************************************************************/
 
 // notes based on test programs results, verified on PCB:
-// - bonkadv_mcu_4_30 is the only cmd that ouputs an odd (655) number of bytes
+// - bonkadv_mcu_4_30 is the only cmd that outputs an odd (655) number of bytes
 // - bonkadv_mcu_4_33: the 32 'zeroed' bytes every 64 bytes are effectively written by the MCU
 
 // MCU executed command: 4300 0100 - factory settings
Index: mame-0.142/src/osd/sdl/man/mame.1
===================================================================
--- mame-0.142.orig/src/osd/sdl/man/mame.1	2011-06-16 09:22:39.000000000 +0200
+++ mame-0.142/src/osd/sdl/man/mame.1	2011-06-16 09:33:13.000000000 +0200
@@ -885,7 +885,7 @@
 .\" +++++++++++++++++++++++++++++++++++++++++++++++++++++++
 .TP
 .B \-keymap
-Enable keymap for non\-QWERTY keyboards. Used in conjuction with
+Enable keymap for non\-QWERTY keyboards. Used in conjunction with
 \fB\-keymap_file\fR.
 .TP
 .B \-keymap_file \fIkeymap_file\fR
Index: mame-0.142/src/osd/sdl/output.c
===================================================================
--- mame-0.142.orig/src/osd/sdl/output.c	2011-06-16 09:22:40.000000000 +0200
+++ mame-0.142/src/osd/sdl/output.c	2011-06-16 09:33:13.000000000 +0200
@@ -82,13 +82,13 @@
 	if (fildes < 0)
 	{
 		output = NULL;
-		mame_printf_verbose("ouput: unable to open output notifier file %s\n", SDLMAME_OUTPUT);
+		mame_printf_verbose("output: unable to open output notifier file %s\n", SDLMAME_OUTPUT);
 	}
 	else
 	{
 		output = fdopen(fildes, "w");
 
-		mame_printf_verbose("ouput: opened output notifier file %s\n", SDLMAME_OUTPUT);
+		mame_printf_verbose("output: opened output notifier file %s\n", SDLMAME_OUTPUT);
 		fprintf(output, "MAME " PID_FMT " START %s\n", osd_getpid(), machine.system().name);
 		fflush(output);
 	}
@@ -109,7 +109,7 @@
 		fflush(output);
 		fclose(output);
 		output = NULL;
-		mame_printf_verbose("ouput: closed output notifier file\n");
+		mame_printf_verbose("output: closed output notifier file\n");
 	}
 }
 
Index: mame-0.142/src/emu/sound/digitalk.c
===================================================================
--- mame-0.142.orig/src/emu/sound/digitalk.c	2011-06-16 09:34:20.000000000 +0200
+++ mame-0.142/src/emu/sound/digitalk.c	2011-06-16 09:34:28.000000000 +0200
@@ -154,7 +154,7 @@
 
 
 Finally, silent zones are compressed specifically by storing their
-lenghts.
+lengths.
 
 
 Decoding is simpler.  The 128-samples waveform is decoded using the
Index: mame-0.142/src/mame/audio/spiders.c
===================================================================
--- mame-0.142.orig/src/mame/audio/spiders.c	2011-06-16 09:34:44.000000000 +0200
+++ mame-0.142/src/mame/audio/spiders.c	2011-06-16 09:34:51.000000000 +0200
@@ -42,7 +42,7 @@
 	0			// no cap
 };
 
-/* The noice generator consists of two LS164 plus a LS74, so the lenght is 8+8+1 */
+/* The noice generator consists of two LS164 plus a LS74, so the length is 8+8+1 */
 static const discrete_lfsr_desc spiders_lfsr =
 {
 	DISC_CLK_IS_FREQ,
Index: mame-0.142/src/mame/drivers/calomega.c
===================================================================
--- mame-0.142.orig/src/mame/drivers/calomega.c	2011-06-16 09:35:57.000000000 +0200
+++ mame-0.142/src/mame/drivers/calomega.c	2011-06-16 09:36:03.000000000 +0200
@@ -575,7 +575,7 @@
         SSI Poker (v4.0)
 
     - Added a list of official/unofficial games, and their states.
-    - Fixed a bug introduced when extended the color PROM region to eliminate colortable lenght.
+    - Fixed a bug introduced when extended the color PROM region to eliminate colortable length.
     - Other changes/fixes.
     - Updated technical notes.
 
Index: mame-0.142/src/mame/drivers/funworld.c
===================================================================
--- mame-0.142.orig/src/mame/drivers/funworld.c	2011-06-16 09:35:43.000000000 +0200
+++ mame-0.142/src/mame/drivers/funworld.c	2011-06-16 09:35:51.000000000 +0200
@@ -93,7 +93,7 @@
   The hardware was designed to manage 4096 tiles with a size of 8x4 pixels each.
   Also support 4bpp graphics and the palette limitation is 8 bits for color codes (256 x 16colors).
   It means the hardware was designed for more elaborated graphics than Jolly Card games...
-  Color PROMs from current games are 512 bytes lenght, but they only use the first 256 bytes.
+  Color PROMs from current games are 512 bytes length, but they only use the first 256 bytes.
 
   Normal hardware capabilities:
 
Index: mame-0.142/src/mame/drivers/magicfly.c
===================================================================
--- mame-0.142.orig/src/mame/drivers/magicfly.c	2011-06-16 09:35:27.000000000 +0200
+++ mame-0.142/src/mame/drivers/magicfly.c	2011-06-16 09:35:36.000000000 +0200
@@ -174,7 +174,7 @@
 
         ($0D)            ; Incremented each time a NMI is triggered.
         ($1D)            ; In case of 0x00, NMI do nothing and return.
-        ($11)            ; Store lenghts for text handling.
+        ($11)            ; Store lengths for text handling.
         ($12)            ; Store values to be written in color RAM.
         ($13 - $14)      ; Pointer to text offset.
         ($15 - $16)      ; Pointer to video RAM.
Index: mame-0.142/src/mame/drivers/snookr10.c
===================================================================
--- mame-0.142.orig/src/mame/drivers/snookr10.c	2011-06-16 09:36:07.000000000 +0200
+++ mame-0.142/src/mame/drivers/snookr10.c	2011-06-16 09:36:17.000000000 +0200
@@ -66,7 +66,7 @@
     8 bits for color codes (256 x 16 colors). It means the hardware was designed for more
     elaborated graphics than these games...
 
-    Color PROMs from current games are 512 bytes lenght, but they only use the first 256 bytes.
+    Color PROMs from current games are 512 bytes length, but they only use the first 256 bytes.
 
     The sound is composed by 4-bit ADPCM samples. All the supported games have the same sound ROM.
     All the sounds/samples were ripped from the Gottlieb pinball 'Cue Ball Wizard'(1992).
@@ -331,7 +331,7 @@
     - Added the oki6295 emulation to all games.
     - Hooked output ports.
     - Documented and calculated all bits related to lamps.
-    - Adjusted palette lenght to 256 colors.
+    - Adjusted palette length to 256 colors.
     - Totally decrypted the apple10 color matrix. Now colors are perfect.
     - Created a new machine driver for apple10 due to encryption.
     - Reverse engineering the code to complete the DIP switches.
Index: mame-0.142/src/mame/video/ddribble.c
===================================================================
--- mame-0.142.orig/src/mame/video/ddribble.c	2011-06-16 09:35:02.000000000 +0200
+++ mame-0.142/src/mame/video/ddribble.c	2011-06-16 09:35:16.000000000 +0200
@@ -176,10 +176,10 @@
 
 ***************************************************************************/
 
-static void draw_sprites( running_machine& machine, bitmap_t *bitmap, const rectangle *cliprect, UINT8* source, int lenght, int gfxset, int flipscreen )
+static void draw_sprites( running_machine& machine, bitmap_t *bitmap, const rectangle *cliprect, UINT8* source, int length, int gfxset, int flipscreen )
 {
 	gfx_element *gfx = machine.gfx[gfxset];
-	const UINT8 *finish = source + lenght;
+	const UINT8 *finish = source + length;
 
 	while (source < finish)
 	{
