Index: mame-0.176/3rdparty/rapidjson/include/rapidjson/rapidjson.h
===================================================================
--- mame-0.176.orig/3rdparty/rapidjson/include/rapidjson/rapidjson.h
+++ mame-0.176/3rdparty/rapidjson/include/rapidjson/rapidjson.h
@@ -250,7 +250,7 @@
 
 //! Whether using 64-bit architecture
 #ifndef RAPIDJSON_64BIT
-#if defined(__LP64__) || defined(_WIN64) || defined(__EMSCRIPTEN__)
+#if defined(__LP64__) || (defined(__x86_64__) && defined(__ILP32__)) || defined(_WIN64) || defined(__EMSCRIPTEN__)
 #define RAPIDJSON_64BIT 1
 #else
 #define RAPIDJSON_64BIT 0
